### PR TITLE
Script configuration need to be in /etc/perun/X_Y.d

### PR DESCRIPTION
**Pull request by stavamichal**
- change all characters '-' in the config path to '_'
 - changed in Makefile by regex
 - changed in rpm by 'tr' command

### CODE REVIEW

File: slave-new/generate_rpm.sh [hunk 1]
```diff
@@ -62,9 +63,9 @@ CUSTOM_CONF=""
 CUSTOM_FILE_DATA=""
 # conf predefined settings
 if [ $WITH_CONF == 1 ]; then
-	CUSTOM_CONF="mkdir -p %{buildroot}/etc/perun/${SERVICE_NAME}.d
-cp -r conf/* %{buildroot}/etc/perun/${SERVICE_NAME}.d"
-	CUSTOM_FILE_DATA="/etc/perun/${SERVICE_NAME}.d"
+	CUSTOM_CONF="mkdir -p %{buildroot}/etc/perun/${CONF_SERVICE_NAME}.d
+cp -r conf/* %{buildroot}/etc/perun/${CONF_SERVICE_NAME}.d"
```
Comment (licehammer) [inline slave-new/generate_rpm.sh]: Any reason why this line is not indented?

**Comment by licehammer** [comment]
Please rebase and also rewrite the commit message to be understandable without known context.

**Comment by stavamichal** [comment]
Done.

**Comment by licehammer** [comment]
Merged as af9da05135a434b9c08e180cdf4bda2dd721cdf7

**Pull request closed without merging**